lib/net/http_fetch_plugin.js

```diff
@@ -290,4 +290,8 @@ if (shaka.net.HttpFetchPlugin.isSupported()) {
       'https', shaka.net.HttpFetchPlugin.parse,
       shaka.net.NetworkingEngine.PluginPriority.PREFERRED,
       /* progressSupport= */ true);
+  shaka.net.NetworkingEngine.registerScheme(
```

Comment:
Can you please move this change to a separate PR?  That way, we can cherry-pick this PR to the older release branches to document the process for them, without importing this new feature.  Thanks!